src/puppetlabs/trapperkeeper/services/metrics/metrics_core.clj

```diff
@@ -35,18 +36,18 @@
 
 (schema/defn initialize :- MetricsServiceContext
   [config :- MetricsConfig]
-  (let [enabled         (ks/to-bool (:enabled config))
+  (let [enabled-given   (contains? config :enabled)
         jmx-config      (get-in config [:reporters :jmx])]
-    (if-not enabled
-      {:registry nil}
-      (let [registry (MetricRegistry.)]
-        (cond-> {:registry registry}
+    (if enabled-given
```

Comment:
Might be better to change this to a `when`, since this `if` doesn't have a false branch.